.github/workflows/integrate.yml

```diff
@@ -0,0 +1,71 @@
+name: Run PHPUnit and PHPCS
```

Comment:
As per https://github.com/Parsely/wp-parsely/tree/develop/.github/workflows, moving linting (`php -l` and running PHPCS, etc.) to a separate workflow file allows that to be run in parallel to running tests, and for only a single version of PHP, instead of repeating multiple lint checks for different versions.